ledger/src/bigtable_upload.rs

```diff
@@ -162,13 +162,13 @@ pub async fn upload_confirmed_blocks(
 
         (
             (0..config.num_blocks_to_upload_in_parallel)
-                .map(|_| {
+                .map(|ix| {
```

Comment:
```suggestion
                .map(|i| {
```
Nit. While I wouldn't be surprised if there are exceptions, we usually use "i" for "index", since "ix" also means "Instruction"

Reply:
Good point - https://github.com/solana-labs/solana/pull/30206 (which I added you as reviewer on) at least adds consistency for thread pools